### PR TITLE
add pointwise benchmark

### PR DESCRIPTION
**Pull request by liqiangxl**
**What is in this PR?**
add a simple pointwise benchmark
```
def simple_pointwise_fwd_fn(inputs: list):  # in_tensor
    return torch.mul(inputs[0], inputs[0])
```
**Why need to add this?**
(1) serve as a basline to check the best bandwidth we can expect for a specific problem size
(2) check whether the device is healthy.

**Current performance:** see [link](https://docs.google.com/document/d/1FtIU_s-iw_Q9C7psdJn5FMp9ApKdzqdkJe4p6R9TQkU/edit?usp=sharing).

### CODE REVIEW

File: benchmarks/python/test_simple_pointwise.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+# SPDX-License-Identifier: BSD-3-Clause
+import pytest
+from nvfuser import FusionDefinition, DataType
+from nvfuser.pytorch_utils import torch_dtype_to_nvfuser_dtype
+from .core import run_benchmark, clear_cuda_cache
+import torch
+from .global_params import generate_input_sizes, FLOAT_DTYPES, PROMOTE_DTYPES
+
+
+def simple_pointwise_fusion(
```
Comment (Priya2698) [inline benchmarks/python/test_simple_pointwise.py:12]: nit:
```Suggestion
def pointwise_mul_fusion(
Similarly for other names in the file

File: benchmarks/python/test_simple_pointwise.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+def test_simple_pointwise_nvf_benchmark(
+    benchmark,
+    size: tuple,
+    dtype: torch.dtype,
+    disable_validation: bool,
+    disable_benchmarking: bool,
+):
+    clear_cuda_cache()
+
+    inputs = [torch.randn(*size, device="cuda", dtype=dtype)]
```
Comment (Priya2698) [inline benchmarks/python/test_simple_pointwise.py:42]: `*size` -> `size`

**Review by Priya2698 — APPROVED** [review]
LGTM. Only minor comments for file and function naming

File: benchmarks/python/test_simple_pointwise.py (hunk 2 above)
Comment (liqiangxl) [reply benchmarks/python/test_simple_pointwise.py:42]: changed.

**Pull request merged**